### PR TITLE
cards redesigned

### PR DESCRIPTION
**Pull request by Eshchock1**
image
image
image
image

### CODE REVIEW

File: components/NotesCard.vue [hunk 1]
```diff
@@ -144,6 +141,25 @@
...
+          :src="note.images[0]"
+          v-if="hasImage && preview"  
+          style="position:absolute;object-fit: cover;min-height:100%"
+        />
+        <img
+          src="../assets/images/Quento_Physics_Filler.png"
+          v-if="!hasImage && preview"  
+          style="position:absolute;object-fit: cover;min-height:100%"
+        />
+
```
Comment (DrCheddar) [inline components/NotesCard.vue:157]: Can we remove the preview image on mobile? 
image
takes up too much space

Comment (DrCheddar) [reply components/NotesCard.vue:157]: Also turn down the max-height on the md preview on mobile? 
image
So either we show an image or md preview not both, saves alot of space

**Review by DrCheddar — CHANGES_REQUESTED** [review]
Overall looks really really good, I think we should do some space optimizations for mobile though to not make each card take up the entire screen. Other than that, just rebase and you are good

File: components/NotesCard.vue (hunk 1 above)
Comment (DrCheddar) [reply components/NotesCard.vue:157]: Also theres this issue with the menu dots 
image
image

**Review by picklepower24 — CHANGES_REQUESTED** [review]
Looks good to me, code is neat. I agree with Jeffrey though, pictures take too much space on mobile.
image
Also, just an idea: maybe have different default images for each course? looks like the current green one could be for chem.

**Comment by Eshchock1** [comment]
image
image

**Review by picklepower24 — DISMISSED** [review]
Some images seem to be broken on older notes, but I like the new image height.
image

**Review by DrCheddar — APPROVED** [review]
Looks fucking good

**Pull request merged**